### PR TITLE
Added regex stresstest to tracks

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: py/tracks/latest.toml [hunk 1]
```diff
@@ -13,6 +13,7 @@ fixtures = [
 queries = [
     "../specs/group_by.toml",
     "../specs/like.toml",
+    "../specs/regex.toml",
```
Comment (mfussenegger) [inline py/tracks/latest.toml]: if this is listed under queries it needs to have the same setup/teardown instructions as `../specs/group_by`.

Comment (matriv) [inline py/tracks/latest.toml]: @mfussenegger Could you please explain? 
For regex.toml I've created a new SQL: sql/uservisits_analyzer.sql because I used an analyzer: `INDEX uagent_idx USING FULLTEXT("UserAgent") with (analyzer='english')`

Do you think I could use this index in uservisits.sql and get rid of this new file?

Comment (mfussenegger) [inline py/tracks/latest.toml]: the files listed under `queries` will use setup/teardown from the file listed under `fixtures`. So in order for this to work correctly the setup/teardown of `regex.toml` must match the one of `group_by.toml`

You can either move `regex.toml` into `full` or adapt the setup so that it matches.

Not sure about the analyzer stuff - why is that needed?

Comment (matriv) [inline py/tracks/latest.toml]: @mfussenegger Without the analyzer the query:
`select * from userVisits where "UserAgent" ~ '(?i).*\bWindows\sNT\s6.0\b.*' limit 10000` always return 0 rows, I'm not sure why I'm not so much into the indexes :-(
